packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -171,12 +232,104 @@ class ScaffoldState extends State<Scaffold> {
     final List<LayoutId>children = new List<LayoutId>();
     _addIfNonNull(children, materialBody, _Child.body);
     _addIfNonNull(children, paddedToolBar, _Child.toolBar);
-    _addIfNonNull(children, config.bottomSheet, _Child.bottomSheet);
+
+    if (_currentBottomSheet != null ||
+        (_dismissedBottomSheets != null && _dismissedBottomSheets.isNotEmpty)) {
+      List<Widget> bottomSheets = <Widget>[];
+      if (_dismissedBottomSheets != null && _dismissedBottomSheets.isNotEmpty)
```

Comment:
I suppose this could be (_dissmissedBottomSheets?.isNotEmpty ?? false).  Not really clearer though.